data.sql

```diff
@@ -11,3 +12,15 @@ INSERT INTO animals VALUES (8, 'Angemon', '12-Jul-2005', 1, 45, TRUE);
 INSERT INTO animals VALUES (9, 'Boarmon', '7-Jun-2005', 7, 20.4, TRUE);
 INSERT INTO animals VALUES (10, 'Blossom', '13-Oct-1998', 3, 17, TRUE);
 INSERT INTO animals VALUES (11, 'Ditto', '14-May-2022', 4, 22, TRUE);
+
+/* owners */
+INSERT INTO owners(full_name, age) VALUES('Sam Smith', 34);
+INSERT INTO owners(full_name, age) VALUES('Jennifer Orwell', 19);
+INSERT INTO owners(full_name, age) VALUES('Bob', 45);
+INSERT INTO owners(full_name, age) VALUES('Melody Pond', 77);
+INSERT INTO owners(full_name, age) VALUES('Dean Winchester', 14);
+INSERT INTO owners(full_name, age) VALUES('Jodie Whittaker', 38);
+
```

Comment:
- [ ] It would be better if we update our animals table to meet these requirements as follows; 

1. If an animal's name ends with mon it should have Digimon as a species_id, all other animals should have Pokemon instead
2. All animals got assigned owners
Kindly move the Update actions into the data.sql file, although I see it in the screenshot

Reply:
Hi @dodoburner , sorry for the misunderstanding, what I mean is that the `UPDATE` you did in the screenshot
```
UPDATE animals
   SET species_id = (SELECT id from species WHERE name = 'Digimon')
   WHERE name like '%mon';
```
Like the above should rather be in the `data.sql` file not the screenshot as we have it. I hope this is clearer now. Kindly let me know if you still don't understand what I mean

Reply:
Alright thanks! @addod19